### DISCUSSION TITLE
"The symbolic link targets a file which does not exist within your site's repository" - but it does!

### DISCUSSION THREAD

**Discussion by activescott** [community/community#21790] [Other Feature Feedback, Questions, & Ideas]
<p>I am running into the common <em>Your site is having problems building: The symbolic link /blue.css targets a file which does not exist within your site’s repository.</em></p>
<p>I know this is a common issue and have read the <a href="https://help.github.com/articles/page-build-failed-symlink-does-not-exist-within-your-site-s-repository/." rel="nofollow noopener">linked documentation</a> carefully. The issue is that the symlink it is referring to in the repo at <a href="https://github.com/activescott/blogs.pingpoet.com/blob/master/docs/blue.css" rel="nofollow noopener">https://github.com/activescott/blogs.pingpoet.com/blob/master/docs/blue.css</a> and <a href="https://github.com/activescott/blogs.pingpoet.com/blob/master/httrack/blogs.pingpoet.com/blue.css" rel="nofollow noopener">the linked target</a> is indeed in the repository. In fact, if you hit that link from the github pages published site at <a href="http://blogs.pingpoet.com/blue.css" rel="nofollow noopener">http://blogs.pingpoet.com/blue.css</a>, it even works!</p>
<p>Am I missing something?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Sorry it has taken us so long to get back to you on this one. Would you mind reaching out to our private support so they can take a deeper look at things to ensure everything is configured correctly? You can reach GitHub Support at our <a href="https://github.com/contact" rel="nofollow noopener">contact form</a>.</p>

**Comment by Octopirate1** [comment]
<p>Hey, thanks for using a public forum to tell someone to message support privately. That’s really what public forms are for.</p>
